src/test/java/seedu/mindmymoney/command/UpdateCommandTest.java

```diff
@@ -25,17 +31,19 @@ void updateCommand_updateExpenditure_listUpdated() {
         UpdateCommand updateCommand = new UpdateCommand(input, testUser);
         try {
             updateCommand.executeCommand();
-            assertEquals(testUser.getExpenditureListArray().get(0), newExpenditure);
+            assertEquals(testUser.getExpenditureListArray().get(0).toString(), newExpenditure.toString());
         } catch (MindMyMoneyException e) {
             System.out.println(e.getMessage());
             fail();
         }
     }
 
-    /** Assert that an invalid command will throw an exception. */
+    /**
+     * Assert that an invalid update expenditure command will throw an exception.
+     * */
```

Comment:
Refine javadoc format
```suggestion
     **/
```